src/main/java/org/opensearch/ad/settings/ADNumericSetting.java

```diff
@@ -37,6 +39,11 @@ public class ADNumericSetting extends DynamicNumericSetting {
                 CATEGORY_FIELD_LIMIT,
                 Setting.intSetting(CATEGORY_FIELD_LIMIT, 2, 0, 5, Setting.Property.NodeScope, Setting.Property.Dynamic)
             );
+            // JVM heap usage threshold setting
+            put(
+                JVM_HEAP_USAGE_THRESHOLD,
+                Setting.intSetting(JVM_HEAP_USAGE_THRESHOLD, 95, 0, 98, Setting.Property.NodeScope, Setting.Property.Dynamic)
```

Comment:
Quick question: How did we decide these values?

Reply:
discussed this topic offline